app/controllers/admin/confirms_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Admin
+  class ConfirmsController < BaseController
```

Comment:
This is a little nitpicky, but you asked for rails feedback...

I would probably put this action either at:

- `Admin::UsersController#update` - with a route like /admin/users/123 - and make sure the only thing which can be updated is the confirmed_at; or
- `Admin::Confirmations#create` with a route like /admin/users/123/confirmation - and have that find that user and confirm it.